cmd/loadtest/loadtest.go

```diff
@@ -759,66 +758,10 @@ func getERC721Contract(ctx context.Context, c *ethclient.Client, tops *bind.Tran
 	return
 }
 
-func blockUntilSuccessful(ctx context.Context, c *ethclient.Client, f func() error) error {
-	numberOfBlocksToWaitFor := *inputLoadTestParams.ContractCallNumberOfBlocksToWaitFor
-	blockInterval := *inputLoadTestParams.ContractCallBlockInterval
-	start := time.Now()
-	currStartBlockNumber, err := c.BlockNumber(ctx)
-	if err != nil {
-		log.Error().Err(err).Msg("Error getting block number")
-		return err
-	}
-	log.Trace().
-		Uint64("currStartBlockNumber", currStartBlockNumber).
-		Uint64("numberOfBlocksToWaitFor", numberOfBlocksToWaitFor).
-		Uint64("blockInterval", blockInterval).
-		Msg("Starting blocking loop")
-	var lastBlockNumber, currentBlockNumber uint64
-	var lock bool
-	for {
-		select {
-		case <-ctx.Done():
```

Comment:
Quick question, does it handle well stopping the loadtest with `CTRL+C`?

Reply:
I've switched to a cancelled